lib/iStats/cpu.rb

```diff
@@ -27,10 +26,15 @@ def all
 
       # Print CPU temperature with sparkline
       #
-      def cpu_temperature
-        t = get_cpu_temp
-        thresholds = [50, 68, 80, 90]
-        puts "CPU temp: #{t}#{Symbols.degree}C  " + Printer.gen_sparkline(t, thresholds)
+      def cpu_temperature        
```

Comment:
I'd rather keep this function to only print temperature from 1 sensor. Perhaps it could be made configurable with the settings you added if the default doesn't work. The flow would be something like:
1. Scan (get list of sensors)
2. `istats set cpu [sensor_key]`

This way the output would still be nice and sweet while allowing custom settings.